src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep.java

```diff
@@ -75,20 +80,44 @@
         return bindings;
     }
 
-    public static final class Execution extends AbstractStepExecutionImpl {
+    @Override
+    public StepExecution start(StepContext context) throws Exception {
+        return new Execution(this, context);
+    }
+
+    public static final class Execution extends StepExecution {
```

Comment:
For running-build compatibility you must continue to extend `AbstractStepExecutionImpl`; see the Javadoc for nondeprecated constructor.